src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,37 @@
+package duke.task;
+
+import duke.Duke;
+
+public class Deadline extends Task {
+    protected String deadline;
+
+
+    public Deadline(String description, String deadline, boolean isFromFile) {
+        super(description);
+        this.deadline=deadline;
+        if(!isFromFile) {
+            printDeadlineMsg(description);
+        }
+    }
+
+    @Override
+    public String toString() {
+        return ("[D]"+"["+ this.getStatusIcon()+ "] "
+                + this.description+" (by: "+this.deadline+")" );
+    }
+    @Override
+    public String toStringFile(){
+        int isDoneInt = isDone? 1:0;
+        return ("D | "+ isDoneInt + " | "
+                + this.description +" | "+this.deadline);
+    }
+
+    public void printDeadlineMsg(String deadline) {
+        String echo_msg ="          ____________________________________________________________\n" +
```

Comment:
Variable name should be in camel case and not have underscore. Also, the name `echo_msg` is a bit strange since it is not really related to the word echo.